src/chat/chatReducers.js

```diff
@@ -14,7 +14,7 @@ import { homeNarrow, isMessageInNarrow } from '../utils/narrow';
 import chatUpdater from './chatUpdater';
 
 const getInitialState = () => ({
-  fetching: { older: false, newer: false },
+  fetching: { older: true, newer: true },
```

Comment:
Since we always do an initial fetch, and the code for initial fetch does not do an action call.